fe1-web/src/features/rollCall/components/__tests__/__snapshots__/EventRollCall.test.tsx.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`EventRollCall should correctly render 1`] = `
+Array [
+  <Text>
+    Roll Call
+  </Text>,
+  <Text>
+    Not Open yet
+  </Text>,
+  <Text>
+    Be sure to have set up your Wallet
```

Comment:
I think in the future it would be nice having a test for rendering the component multiple times. Once with the wallet not being setup, once with the wallet being setup and the roll call not being open yet, etc. :)

Reply:
I'll think about that